tsconfig.base.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "module": "NodeNext",
+    "module": "CommonJS", // DO NOT CHANGE: https://github.com/web-infra-dev/rspack/pull/4650
```

Comment:
@Boshen why not keep the latest TS version and just leave this as CommonJS?

Reply:
The latest version threw an error, you can give it a try though.